src/scene_battle_rpg2k3.cpp

```diff
@@ -924,11 +932,17 @@ bool Scene_Battle_Rpg2k3::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBas
 				Sprite_Battler::LoopState_WaitAfterFinish);
 		}
 
+		if (Game_System::GetInvertAnimations()) {
+			invert_animation = (action->GetSource()->IsDirectionFlipped() ^ (action->GetSource()->GetType() == Game_Battler::Type_Enemy));
```

Comment:
Storing state in the scene adds complexity, makes the code harder to follow, and makes it easier to introduce bugs. Instead, I would suggest re-computing this boolean flag directly using the source and target instead of storing and retrieving this `invert_animation` flag.